### DISCUSSION TITLE
Change Certificate CN from existing "Composer_GarryJailbreak"

### DISCUSSION THREAD

**Discussion by ThePopolou** [garrynewman/Control4.Jailbreak#33] [Ideas]
Hello

I've noticed that the CN for the certificate that the JB creates is `Composer_GarryJailbreak`. Is this wise? The Director logs are constantly filled with this whenever a connection is made using this certificate which you can imagine is false economy if they are being sent to Snap or if the logs are ever inspected.

Could this be a user-selectable CN or something else chosen that should obfuscate the fact that a JB was used please.

Thanks

**Comment by BCSugoi** [comment]
Hello ThePopolou...  I had the same concern as you so I downloaded Garry's project and figured out how to feed the parameters in with a configuration file, config.xml.   You can check out my "version 6 beta" here:  https://github.com/BCSugoi/Control4.JailbreakBCSugoi/releases/tag/Beta

**Reply by ThePopolou** [reply]
Love what you did with it; works very well indeed. My initial solution was a little crude at first but as a proof it worked, so had begun drafting up something fairly similar to the route you took.

**Comment by garrynewman** [comment]
What would you change it to?

**Reply by BCSugoi** [reply]
I'm just using a random email address,   following the pattern used by the dealer's employees. eg. something like "Composer_gwilson@acmesystems.com" would make the log entries for my changes blend right in with those logged by the dealer's staff.  I think every user is going to have their own views about what they are comfortable seeing in the logs, so it's best to let user set the name at runtime.
